net-im/telegram-desktop/telegram-desktop-4.3.1.ebuild

```diff
@@ -49,7 +49,6 @@ RDEPEND="
 	media-video/ffmpeg:=[opus]
 	sys-libs/zlib:=[minizip]
 	x11-libs/xcb-util-keysyms
-	virtual/opengl
```

Comment:
What's up with this? The program didn't launch for me before I installed *virtual/opengl*, and then it launched and asked whether I wanted to disable opengl but it didn't allow me to.

Reply:
An unintentional change, sorry